File: java/repository/src/main/java/validators/database/TeamDatabaseValidator.java
package validators.database;

import models.Team;
import validators.interfaces.TeamValidator;
import validators.interfaces.Validator;

public class TeamDatabaseValidator implements TeamValidator {
    @Override
    public void validate(Team model) {
        if (model.getTeamName().equals(""))
            throw new Validator.InvalidModelException("Team name can't be empty!");
    }
}

File: csharp/Repository/Validators/Database/TeamDatabaseValidator.cs
using Domain.Models;
using Repository.Interfaces;
using Repository.Validators.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Repository.Validators.Database.Postgresql
{
    public class TeamDatabaseValidator : ITeamValidator
    {
        public void Validate(Team model)
        {
            if (model.TeamName.Equals(""))
                throw new ITeamValidator.InvalidModelException("Team name can't be empty!");
        }
    }
}
